sample_attributes.yml

```diff
@@ -4,9 +4,10 @@ registry_cert_path: /etc/docker/certs.d
 registry_name: /etc/docker/certs.d/registry_hostname:port
 registry_ca_file: /etc/docker/certs.d/registry_hostname:port/ca.crt
 container_user: vagrant
-container_capadd: null
+container_capadd: ''
 authorization_plugin: authz-broker
 log_driver: syslog
 log_opts: /syslog-address/
 app_armor_profile: docker-default
 selinux_profile: /label\:level\:s0-s0\:c1023/
+benchmark_version: 1.12.0
```

Comment:
Should we make the a default too?

Reply:
will do so, closing this pr and creating a new one...

Reply:
You can also force-push in your current branch and github updates the PR

Reply:
That did not worik... will create a new one with signoff